app/src/main/java/com/example/dolbomi/LifePatternActivity.java

```diff
@@ -66,17 +87,803 @@ protected void onCreate(Bundle savedInstanceState) {
+        }
+        if (hour_5 >= 24) {
+            hour_5 -= 24;
+            day_5 += 1;
+        }
+        if(day_5 < 10) {
+            day_s_5 = "0" + day_5;
+        } else {
+            day_s_5 = String.valueOf(day_5);
+        }
+
+        date6.add(Calendar.DATE, -6);
+        int day_6 = date6.get(Calendar.DAY_OF_MONTH);
+        int hour_6 = date1.get(Calendar.HOUR_OF_DAY) + 9;
+        // 전 달로 넘어갈 경우
+        int month_6;
+        if (day_6 > day_ && flag == 0) {
+            month_6 = month_ - 1;
+            month_ = month_6;
+            if(month_6 < 10) {
+                month_s_6 = "0" + month_6;
+            } else {
+                month_s_6 = String.valueOf(month_6);
+            }
+            flag = 1;
+        } else {
+            month_6 = month_;
+            if(month_6 < 10) {
+                month_s_6 = "0" + month_6;
+            } else {
+                month_s_6 = String.valueOf(month_6);
+            }
+        }
+        if (hour_6 >= 24) {
+            hour_6 -= 24;
+            day_6 += 1;
+        }
+        if(day_6 < 10) {
+            day_s_6 = "0" + day_6;
+        } else {
+            day_s_6 = String.valueOf(day_6);
+        }
+
+        date7.add(Calendar.DATE, -7);
+        int day_7 = date7.get(Calendar.DAY_OF_MONTH);
+        int hour_7 = date1.get(Calendar.HOUR_OF_DAY) + 9;
+        // 전 달로 넘어갈 경우
+        int month_7;
+        if (day_7 > day_ && flag == 0) {
+            month_7 = month_ - 1;
+            month_ = month_7;
+            if(month_7 < 10) {
+                month_s_7 = "0" + month_7;
+            } else {
+                month_s_7 = String.valueOf(month_7);
+            }
+            flag = 1;
+        } else {
+            month_7 = month_;
+            if(month_7 < 10) {
+                month_s_7 = "0" + month_7;
+            } else {
+                month_s_7 = String.valueOf(month_7);
+            }
+        }
+        if (hour_7 >= 24) {
+            hour_7 -= 24;
+            day_7 += 1;
         }
+        if(day_7 < 10) {
+            day_s_7 = "0" + day_7;
+        } else {
+            day_s_7 = String.valueOf(day_7);
+        }
+
 
         todayDate = year_ + "-" + month_s + "-" + day_s;
+        todayDate_1 = year_ + "-" + month_s_1 + "-" + day_s_1;
+        todayDate_2 = year_ + "-" + month_s_2 + "-" + day_s_2;
+        todayDate_3 = year_ + "-" + month_s_3 + "-" + day_s_3;
+        todayDate_4 = year_ + "-" + month_s_4 + "-" + day_s_4;
+        todayDate_5 = year_ + "-" + month_s_5 + "-" + day_s_5;
+        todayDate_6 = year_ + "-" + month_s_6 + "-" + day_s_6;
+        todayDate_7 = year_ + "-" + month_s_7 + "-" + day_s_7;
 
 
 
         getLifePatternValue();
 
+        getLifePatternValue_1();
+
+        getLifePatternValue_2();
+
+        getLifePatternValue_3();
+
+        getLifePatternValue_4();
+
+        getLifePatternValue_5();
+
+        getLifePatternValue_6();
+
+        getLifePatternValue_7();
+
+        getTodayLifePatternValue();
```

Comment:
```suggestion
       for(int i=0; i<7; i++){
            getLifePatternValue1to7();
        }        
        getTodayLifePatternValue();
    }
    synchronized private void getLifePatternValue1to7(){
        int count = 0;
        switch(count){
            case 0: getLifePatternValue_1();
                    count++;
                    break;
            case 1: getLifePatterValue_2();
            .
            .
            .
       }
```
처럼 `synchronized`키워드를 사용해보시는 것은 어떨까요?

Reply:
안되네요...